aspnetcore/signalr/diagnostics.md

```diff
@@ -199,6 +199,39 @@ You can attach Diagnostics files to GitHub issues by renaming them so they have
 
 ![Dragging log files on to a GitHub issue](diagnostics/attaching-diagnostics-files.png)
 
+## Metrics
+
+Metrics is a representation of data measures over intervals of time. For example, requests per second. Metrics data allows observation of the state of an app at a high level. .NET gRPC metrics are emitted using <xref:System.Diagnostics.Tracing.EventCounter>.
+
+### SignalR server metrics
+
+SignalR server metrics are reported on the <xref:Microsoft.AspNetCore.Http.Connections> event source.
```

Comment:
`<xref:Microsoft.AspNetCore.Http.Connections>`

This is a name not a type reference in the case of event source